### ISSUE TITLE
Support for multiple gradle tasks

### ISSUE THREAD

**Issue by ZakTaccardi** [fastlane/fastlane#14384]
### Feature Request
API support for working with parallel execution of multiple gradle tasks

#### Motivation Behind Feature
Faster builds

#### Feature Description
This is a duplicate of https://github.com/fastlane/fastlane/issues/4702 that was prematurely closed. I'd rather this issue be closed, and reopen #4702

Fastlane has a special API for running a single gradle task. What's the recommend approach for the existing API to support execution of multiple gradle tasks?

```ruby
gradle(
  task: 'clean'
)
gradle(
  task: 'assemble',
  build_type: 'Release'
)
gradle(
  task: 'testReleaseUnitTest'
)
```

Should a new API be created to run multiple tasks at once?

#### Alternatives or Workarounds
```ruby
gradle(
  task: 'clean assembleRelease testReleaseUnitTest'
)
```

**Comment by joshdholtz** [comment]
@ZakTaccardi Heyy 👋 Thanks for bringing this up! 

>API support for working with parallel execution of multiple gradle tasks

I don't think #4702 wasn't meant to run things in parallel to make it faster. I think that issue was meant that to speed things up linearly but by all running under the same gradle single gradle process instead of multiples.

## Thoughts
It is a bit unclear right now what the recommended approach is for running multiple grade tasks 🤔 I personally like your workaround of...
```rb
gradle(
  task: 'clean assembleRelease testReleaseUnitTest'
)
```
I think that looks the cleanest and is also a lot cleaner than the suggested new `tasks` option mentioned in #4702

One benefit of having breaking up into multiple `gradle` commands is it gives you the ability to see what failed a little bit easier within the _fastlane_ output but this approach does, however, possibly make the `Fastfile` a bit messier looking.

I do Android development but I don't usually make a lot of gradle calls like these so I'm not sure what most Android developers (and _fastlane_ users) would find easiest. I think we definitely need to update the docs to show an example of `task: 'clean assembleRelease testReleaseUnitTest'` but it probably is time to add that new `tasks` option so multiple can be run under the same gradle run.

**Comment by NikolayManzhos** [comment]
I think it might be a bit confusing to have both ```task``` and ```tasks``` options available at the same time. Which option should have a higher priority in case both specified? Maybe we should allow ```task``` option to accept an array as well as a string? WDYT @ZakTaccardi @joshdholtz ?

**Comment by mieliespoor** [comment]
So just to make sure, is this work around available now?

    gradle(
         task: 'clean assembleRelease testReleaseUnitTest'
    )

Currently we call gradle using `sh` commands, but would rather use the fastlane action if possible. Our gradle call looks like this for some apps:

    ./gradlew clean assembleRelease testReleaseUnitTest connectedAndroidTest uploadArchives crashlyticsUploadDistributionRelease

**Comment by mieliespoor** [comment]
Bump. Shouldn't be closed because it's still not resolved. Logging a new issue is pointless is this one has been closed without any action.
